gstreamer/gstmedia.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+from gi.repository import Gst as gst
+from gi.repository import GLib
```

Comment:
Alphabetical order

Reply:
This is correct, that's how the formatter tool orders it.